packages/generator/src/connection/__tests__/__snapshots__/ConnectionGenerator.spec.js.snap

```diff
@@ -2,14 +2,30 @@ exports[`test generate a connection 1`] = `
 Object {
   "connection": "// @flow
 
-import { connectionDefinitions } from \'graphql-relay\';
+import { connectionDefinitions } from 'graphql-relay';
 
-import ExampleType from \'../type/ExampleType\';
+import ExampleType from '../type/ExampleType';
 
 export default connectionDefinitions({
-  name: \'Example\',
+  name: 'Example',
```

Comment:
I wonder why this line changed and removed the `\`, same thing happened on the last PR.